Cython/Debugger/Tests/test_libcython_in_gdb.py

```diff
@@ -457,10 +456,10 @@ def test_cy_eval(self):
         self.break_and_run('os.path.join("foo", "bar")')
 
         result = gdb.execute('print $cy_eval("None")', to_string=True)
-        assert re.match(r'\$\d+ = None\n', result), result
+        self.assertRegexpMatches(result, r'^\$\d+ = None\n$')
```

Comment:
Appending the `$` changes the regex.

Reply:
[resolved](https://github.com/cython/cython/pull/4340/commits/7cd4748b7e9422d2a630f6ef902127516c8a0030)